packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -286,6 +287,48 @@ class BannerAd extends MobileAd {
   }
 }
 
+/// A NativeAd for the [FirebaseAdMobPlugin].
+///
+/// Native ads are ad assets that are presented to users via UI components that
+/// are native to the platform. They're shown using the same types of views with
+/// which you're already building your layouts, and can be formatted to match
+/// the visual design of the user experience in which they live. In coding
+/// terms, this means that when a native ad loads, your app receives a NativeAd
+/// object that contains its assets, and the app (rather than the Google Mobile
+/// Ads SDK) is then responsible for displaying them.
+class NativeAd extends MobileAd {
+  NativeAd(
+      {@required String adUnitId,
```

Comment:
nit: use the same formatting as in MobileAd above

Reply:
done https://github.com/FirebaseExtended/flutterfire/pull/169/files#diff-8324a81da49ba02b4e4b58408c3db9cbR314